homeassistant/components/oru/sensor.py

```diff
@@ -0,0 +1,92 @@
+"""Platform for sensor integration."""
+from datetime import timedelta
+import logging
+
+import voluptuous as vol
+
+from oru import Meter
+from oru import MeterError
+
+from homeassistant.components.sensor import PLATFORM_SCHEMA
+import homeassistant.helpers.config_validation as cv
+from homeassistant.const import ENERGY_KILO_WATT_HOUR
+from homeassistant.helpers.entity import Entity
+
+_LOGGER = logging.getLogger(__name__)
+
+CONF_METER_NUMBER = "meter_number"
+
+SCAN_INTERVAL = timedelta(minutes=15)
+
+SENSOR_NAME = "ORU Current Energy Usage"
+SENSOR_ICON = "mdi:counter"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({vol.Required(CONF_METER_NUMBER): cv.string})
+
+
+def setup_platform(hass, config, add_entities, discovery_info=None):
+    """Set up the sensor platform."""
+
+    meter_number = str(config.get(CONF_METER_NUMBER, None))
```

Comment:
Please use `dict[key]` for required config keys and keys with default config schema values.

Reply:
No need to copy to str since the option is validated already by the config schema.

Reply:
Done